pkg/kubernetes/kubernetes.go

```diff
@@ -50,8 +50,13 @@ type InitConfiguration struct {
 	Wait             bool
 	Timeout          uint
 	ImageRegistryURI string
+	ImageVariant     string
 }
 
+var (
+	defaultImageVariantName = "default"
```

Comment:
IMO add a mariner variant constant and verify incoming value matches the constant string ... otherwise error out ...

Reply:
Agree. Having default value as "default" does seem a bit off.

Reply:
Done